velox/common/file/FileSystems.cpp

```diff
@@ -117,9 +117,4 @@ void registerLocalFileSystem() {
       LocalFileSystem::schemeMatcher(), LocalFileSystem::fileSystemGenerator());
 }
 
-// Register FileSystems.
-void registerAllFileSystems() {
```

Comment:
nit: perhaps, submit this change as a separate PR. This will make this one a bit smaller and easier to review.

Reply:
done